.organizationmap

```diff
@@ -339,6 +339,7 @@ Roald van Loon Consultancy <roald@roaldvanloon.nl> Roald J. van Loon <roald@roal
 Ruijie Networks <contact@ruijie.com.cn> Ning Yao <yaoning@ruijie.com.cn>
 Sandia National Laboratories <contact@sandia.gov> Jim Schutt <jaschut@sandia.gov>
 Samsung <contact@samsung.com> James Liu <james.liu@ssi.samsung.com>
+Samsung <contact@samsung.com> Jianjian Huo <samuel.huo@gmail.com>
```

Comment:
here it should be `Samsung <contact@samsung.com> <jianjian.huo@ssi.samsung.com>`, the mailmap will automatically associate your work email as primary